startup/11-optics.py

```diff
@@ -31,9 +31,9 @@ def check_shutters(check, status):
     '''
 
     if check is False:
-        print("\n**************************************************")
-        print("WARNING: Shutters are not controlled in this scan.")
-        print("**************************************************\n")
+        print("\n****************************************************")
+        print(" WARNING: Shutters are not controlled in this scan.")
```

Comment:
Formatting can be automated a bit, such as:
```py
msg = " WARNING: Shutters are not controlled in this scan."
msg_len = len(msg)
print(f"\n{'*' * msg_len}")
print(msg)
print(f"{'*' * msg_len}\n")
```
Result:
```py
In [1]: msg = " WARNING: Shutters are not controlled in this scan."
   ...: msg_len = len(msg)
   ...: print(f"\n{'*' * msg_len}")
   ...: print(msg)
   ...: print(f"{'*' * msg_len}\n")

***************************************************
 WARNING: Shutters are not controlled in this scan.
***************************************************
```

Or, I'd recommend using [`textwrap`](https://docs.python.org/3.7/library/textwrap.html) or [`ansiwrap`](https://pypi.org/project/ansiwrap/) to do it with less typing. We deploy those libs with our standard collection envs, and some beamlines use them (SST, BMM, probably others).